packages/office-ui-fabric-react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -353,6 +353,6 @@ function _isPivotItem(item: React.ReactNode): item is PivotItem {
     !!item &&
     typeof item === 'object' &&
     !!(item as React.ReactElement).type &&
-    ((item as React.ReactElement).type as React.ComponentType).name === PivotItem.name
+    ((item as React.ReactElement).type as any).name === (PivotItem as any).name
```

Comment:
It's so strange that this is where the weird typing issue was popping up, when there's absolutely no connection to Pivot as far as I can tell... Also kind of surprised the issue didn't just move somewhere else. But if this works around it... 🤷‍♀️